scripts/runner.py

```diff
@@ -126,22 +126,27 @@ def load_run_options(cfg):
             'export XRD_RUNFORKHANDLER=1',
             f'export X509_USER_PROXY={_x509_path}',
             # f'export X509_CERT_DIR={os.environ["X509_CERT_DIR"]}',
-            'source /etc/profile.d/conda.sh',
+            'source /etc/profile.d/conda.sh',  # This looks site-specific, may not work everywhere.
```

Comment:
true, we need still to refactor the runner interface to make it more sites-generic.

Reply:
The way to unify this, I think, is to execute `.bashrc`. Then it's the responsibility of the user to setup conda there.